doc/changelog/10-standard-library/10445-constructive-reals.rst

```diff
@@ -0,0 +1,30 @@
+- New module `Reals.ConstructiveCauchyReals` defines constructive real numbers
+  by Cauchy sequences of rational numbers. Classical real numbers are now defined
+  as a quotient of these constructive real numbers, which significantly reduces
+  the number of axioms needed (see `Reals.Rdefinitions` and `Reals.Raxioms`),
+  while preserving backward compatibility.
+
+  Futhermore, the new axioms for classical real numbers include the limited
+  principle of omniscience (`sig_forall_dec`), which is a logical principle
+  instead of an ad hoc property of the real numbers. In a future pull request
+  we propose to deduce the other axiom, `completeness`, from the logical
+  principle `sig_not_dec` (see `Reals.Rlogic`), which is the excluded middle
+  of negations in sort `Type`.
```

Comment:
This last sentence does not belong to the changelog (which is about what is done), you could move it to the commit messages.

Reply:
Done